### DISCUSSION TITLE
Latency certifier minigame runs worse than gameplay

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#18660] [Q&A]
Not sure if that is correct, but it seems that higher Update thread limiter in latency certifier minigame causes my CPU to be utilised to 100%, which in turn cripples the Draw thread fps. This results in a higher total latency than actual gameplay:
![osu_2022-06-12_23-42-52](https://user-images.githubusercontent.com/53346335/173252995-c6fcbddb-dd5b-436b-845a-a6a291783afc.png)

![osu_2022-06-12_23-43-21](https://user-images.githubusercontent.com/53346335/173253006-9d32a29b-8f4e-4184-9374-f81e4e1056b2.png)

Logs:
[performance.log](https://github.com/ppy/osu/files/8886808/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8886809/runtime.log)

**Comment by peppy** [comment]
That's very weird - the draw thread should not be reduced that much from update thread running unlimited. But a good reason to limit the upper rates in this mode as well.
